src/state_machine/sm.cpp

```diff
@@ -224,6 +224,7 @@ BULLSEYE_EXCLUDE_BLOCK_END
 
 	// Run print event info function
 	if (m_new_event_notify_func) {
+		/* coverity[check_return] */
```

Comment:
General question: we have 2 options how to suppress unused return value:

1. Coverity comment. This works only for the Coverity analyzer and other analyzers and compilers still can generate a warning
2. (void)func(); notation. This approach works for all cases and shows that we intentionally ignore optional return value.

We need to decide which option is better.

Reply:
will need to decide with the bigger quorum